packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -179,12 +194,20 @@ async function transformTransaction(tx: Transfer): Promise<MergedTransaction> {
   )
 }
 
+function isTestnetChainPair(chainPair: ChainPair) {
+  return isNetwork(chainPair.parentChain).isTestnet
+}
+
 /**
  * Fetches transaction history only for deposits and withdrawals, without their statuses.
  */
 const useTransactionHistoryWithoutStatuses = (
   address: `0x${string}` | undefined
 ) => {
+  const [isTestnetMode = false] = useLocalStorage<boolean>(
```

Comment:
Make this a hook `useIsTestnetMode`? It's used in multiple places now and we have to do this every time